tests/monotouch-test/ARKit/ARPointCloudTest.cs

```diff
@@ -0,0 +1,99 @@
+﻿//
+// Unit tests for ARPointCloud
+//
+// Authors:
+//	Vincent Dondain <vidondai@microsoft.com>
+//
+// Copyright 2017 Microsoft. All rights reserved.
+//
+
+#if XAMCORE_2_0 && __IOS__
+
+using System;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using ARKit;
+using AVFoundation;
+using Foundation;
+using NUnit.Framework;
+using ObjCRuntime;
+using OpenTK;
+
+namespace MonoTouchFixtures.ARKit {
+
+	class ARPointCloudPoker : ARPointCloud {
+
+		GCHandle vectorArrayHandle;
+		GCHandle identifiersHandle;
+		Vector3 [] vectorArray;
+		ulong [] identifiers;
+
+		public ARPointCloudPoker () : base (IntPtr.Zero)
```

Comment:
^ that `IntPtr.Zero` is not the way to subclass a native type and (likely) why the test failed